src/chart/generator/axis.cpp

```diff
@@ -20,10 +20,12 @@ Geom::Point MeasureAxises::origo() const
 
 MeasureAxis::MeasureAxis(Math::Range<double> interval,
     std::string &&unit,
+    const std::string_view &measName,
     std::optional<double> step) :
     enabled(true),
     range(interval),
     unit(std::move(unit)),
+    origMeasureName(measName),
```

Comment:
What is happens if datatable has 4 measure, we have a plot at least one measure (which use SSO), then we add another measure?